testSetup.js

```diff
@@ -1,5 +1,3 @@
 /* eslint-env node */
 import 'jest-axe/extend-expect';
-import filterXss from './app/assets/javascripts/lib/xss';
-
-global.filterXss = filterXss;
+import './app/assets/javascripts/lib/xss';
```

Comment:
Importing the script will set up `filterXSS` so that it's globally available. setting global.filterXss was incorrect and never required.